backend/routes/users.js

```diff
@@ -1,9 +1,10 @@
 import express from 'express'
 
 import { UsersController } from '../controllers/users.js'
+import { tokenChecker } from '../utils/expresssMiddleware.js'
 
 export const usersRouter = express.Router()
 
-usersRouter.post('/', UsersController.Create)
-
+usersRouter.post('/:user_id/profile', tokenChecker, UsersController.CreateProfile)
 usersRouter.get('/:user_id', UsersController.FindUserById)
```

Comment:
we could do it this way, on my branch PD-26  I've made a file with user route which is protected and another file in which user route isn't protected, rather then adding the token to specific routes in one file.